pkg/gofr/datasource/cassandra/cassandra.go

```diff
@@ -61,14 +61,15 @@ func New(conf Config) *Client {
 }
 
 // Connect establishes a connection to Cassandra and registers metrics using the provided configuration when the client was Created.
-func (c *Client) Connect() {
-	c.logger.Debugf("connecting to Cassandra at %v on port %v to keyspace %v", c.config.Hosts, c.config.Port, c.config.Keyspace)
+func (c *Client) Connect(_ context.Context) error {
+	c.logger.Logf("connecting to cassandra at %v on port %v to keyspace %v", c.config.Hosts, c.config.Port, c.config.Keyspace)
```

Comment:
I think this change is unintended

```suggestion
	c.logger.Logf("connecting to Cassandra at %v on port %v to keyspace %v", c.config.Hosts, c.config.Port, c.config.Keyspace)
```